Gemfile

```diff
@@ -12,6 +12,7 @@ branch = File.read(File.expand_path("../maintenance-branch", __FILE__)).chomp
   end
 end
 
+gem 'coderay' # for syntax highlighting
```

Comment:
It's not related to the changes here other than I wanted syntax highlighting in the failure output while working on this, so I added it, and I think it's worth keeping :).